osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -165,8 +171,11 @@ public void RestoreAll()
         /// <returns>A list of available <see cref="BeatmapSetInfo"/>.</returns>
         public List<BeatmapSetInfo> GetAllUsableBeatmapSets()
         {
-            using (var context = contextFactory.CreateContext())
-                return context.All<BeatmapSetInfo>().Where(b => !b.DeletePending).Detach();
+            return contextFactory.Run(realm =>
+            {
+                realm.Refresh();
```

Comment:
What's the rationale for this explicit refresh? Wasn't there before.

Reply:
see https://github.com/ppy/osu/pull/16535/commits/495636538fc23101ec641bdf66eb0ba8b3f6a88c